src/v2i-hub/CDASimAdapter/scripts/send_timestep_udp.py

```diff
@@ -35,7 +35,7 @@ def generate_time_sync():
         count_num += 1
         sock.sendto(encoded_msg,host)
         print( encoded_msg.decode(encoding= 'UTF-8'), 'was sent to ', host)
-        time.sleep(5)
+        time.sleep(.1)
```

Comment:
Is there a reason why we reduced to .1?  Should we declare this in such a way that it can be configurable?

Reply:
I don't think this is necessary. This is a testing script for testing simulation mode without CDASim. I think leaving it as it is is probably fine. I definitely don't want to spend time addressing it under this PR. 0.1 seconds the interval at which CDASim updates time and so I think this is appropriate for the script.